src/radical/pilot/task_overlay/worker.py

```diff
@@ -467,10 +500,18 @@ def _dispatch(self, task):
 
         # ----------------------------------------------------------------------
         def _dispatch_thread(tlock):
+            # FIXME: do we still need this thread?
+
+            os.environ['RP_TASK_CORES'] = ','.join(str(i) for i in task['resources']['cores'])
+            os.environ['RP_TASK_GPUS']  = ','.join(str(i) for i in task['resources']['gpus'])
```

Comment:
with these two env variables, it is not clear why they are introduced (one of them is used inside in this method as well and nowhere else)

Reply:
Yeah, we probably can remove them.  I used them for testing to ensure that the tasks land on different cores, but otherwise they are likely not useful.